README.md

```diff
@@ -1,36 +1,21 @@
 # Knockout Pickadate Binding
 
-> Simple knockout binding that wraps [pickadate.js](http://amsul.ca/pickadate.js).
+> Simple knockout binding which wraps [pickadate.js](http://amsul.ca/pickadate.js).
```

Comment:
![one does not simply use restrictive clauses all willy-nilly](https://cdn.meme.am/instances/500x/69049464.jpg)